spark/v3.0/spark/src/main/java/org/apache/spark/sql/connector/iceberg/read/SupportsFileFilter.java

```diff
@@ -32,5 +32,22 @@
    *
    * @param locations file locations
    */
-  void filterFiles(Set<String> locations);
+  SupportsFileFilter.FileFilterMetric filterFiles(Set<String> locations);
```

Comment:
What about using a `Pair<Integer, Integer>` instead of defining a new class?

Reply:
I think the current way may be better readable.